script.js

```diff
@@ -0,0 +1,68 @@
+//Set the buttons as const just to keep things a bit cleaner
+const dailyButton = document.querySelector(".daily__button");
+const generalButton = document.querySelector(".general__button");
+const constantButton = document.querySelector(".constant__button");
+
+
+//Event listeners for clicking on the three "add item" buttons or pressing the enter key when the three text boxes are in focus
+dailyButton.addEventListener("click", function () {
+    addToList(".daily__textBox", ".daily__list");
+});
+document.querySelector(".daily__textBox").addEventListener("keydown", function (e) {
+    if (e.key === "Enter") {
+        addToList(".daily__textBox", ".daily__list");
+    }
+});
+
+generalButton.addEventListener("click", function () {
+    addToList(".general__textBox", ".general__list");
+});
+document.querySelector(".general__textBox").addEventListener("keydown", function (e) {
+    if (e.key === "Enter") {
+        addToList(".general__textBox", ".general__list");
+    }
+});
+
+constantButton.addEventListener("click", function () {
+    let listType = chooseConstantList();
+    addToList(".constant__textBox", listType);
+});
+document.querySelector(".constant__textBox").addEventListener("keydown", function (e) {
+    if (e.key === "Enter") {
+        let listType = chooseConstantList();
+        addToList(".constant__textBox", listType);
+    }
+});
+
+
+//The main function to add an item to a list. It checks to make sure that the text box has content before adding to the list to avoid empty items
+function addToList(textNumber, listType){
```

Comment:
Could also change it to textBox and list, and do the querySelector call outside of the function. This feels conceptually cleaner to me.

Reply:
Changed the function in [fec8329](https://github.com/Vhynn/ToDoApp/pull/9/commits/fec83298ff41b7b03d73e78dd5091ac0e1e48424), I went for the second option because that made more sense to me too.